consensus/src/commons.rs

```diff
@@ -139,35 +141,29 @@ impl QuorumMsgSender {
     }
 
     /// Sends an quorum (internally) to the quorum loop.
-    pub(crate) async fn send(&self, msg: Message) -> bool {
-        if let Payload::Quorum(q) = &msg.payload {
-            if q.validation.is_empty()
-                || q.ratification.is_empty()
-                || q.vote == Vote::NoCandidate
+    pub(crate) async fn send(&self, msg: Message) {
+        match &msg.payload {
             // TODO: Change me accordingly to https://github.com/dusk-network/rusk/issues/1268
+            Payload::Quorum(q)
+                if !q.validation.is_empty()
+                    && !q.ratification.is_empty()
+                    && q.vote != Vote::NoCandidate =>
             {
-                return false;
+                tracing::debug!(
+                    event = "send quorum_msg",
+                    vote = %q.vote,
+                    round = msg.header.round,
+                    iteration = msg.header.iteration,
+                    validation = ?q.validation,
+                    ratification = ?q.ratification,
+                );
             }
-
-            tracing::debug!(
-                event = "send quorum_msg",
-                vote = %q.vote,
-                round = msg.header.round,
-                iteration = msg.header.iteration,
-                validation = ?q.validation,
-                ratification = ?q.ratification,
-            );
-
-            let _ = self
-                .queue
-                .send(msg.clone())
-                .await
-                .map_err(|e| error!("send quorum_msg failed with {:?}", e));
-
-            return true;
+            _ => return,
         }
 
-        false
+        if let Err(e) = self.queue.send(msg).await {
+            error!("send quorum_msg failed with {e:?}")
+        }
```

Comment:
At some point we might want to go through all the `Sender`s that we have and check if it is even possible that they error while sending. As a reminder, a `Sender` only errors when `send` is called when the `Receiver` is dropped. If the receiver is never dropped, we might as well use `unwrap()` or `expect()`

Reply:
Receiver is currently dropped when a block is accepted.
That's why we just logging for now, probably we'll lower the level later on